src/doc/de/tutorial/tour_advanced.rst

```diff
@@ -34,6 +34,40 @@ wir diese schneiden und dann die irreduziblen Komponenten berechnen.
 
     sage: V = C2.intersection(C3)
     sage: V.irreducible_components()
+    [Closed subscheme of Affine Space of dimension 2 over Rational Field defined by:
+    y,
+    x - 1,
+    Closed subscheme of Affine Space of dimension 2 over Rational Field defined by:
+    y - 1,
+    x,
+    Closed subscheme of Affine Space of dimension 2 over Rational Field defined by:
+    x + y + 2,
+    2*y^2 + 4*y + 3]
+<<<<<<< HEAD
+    [
+    Closed subscheme of Affine Space of dimension 2 over Rational Field defined by:
+      y - 1,
+      x,
+    Closed subscheme of Affine Space of dimension 2 over Rational Field defined by:
+      y,
+      x - 1,
+    Closed subscheme of Affine Space of dimension 2 over Rational Field defined by:
+      x + y + 2,
+      2*y^2 + 4*y + 3
+    ]
+||||||| c9dd1e85118
+    [
+    Closed subscheme of Affine Space of dimension 2 over Rational Field defined by:
+      y,
+      x - 1,
+    Closed subscheme of Affine Space of dimension 2 over Rational Field defined by:
+      y - 1,
+      x,
+    Closed subscheme of Affine Space of dimension 2 over Rational Field defined by:
+      x + y + 2,
+      2*y^2 + 4*y + 3
+    ]
+=======
     [Closed subscheme of Affine Space of dimension 2 over Rational Field defined by:
```

Comment:
```
    [Closed subscheme of Affine Space of dimension 2 over Rational Field defined by:
      y - 1,
      x,
     Closed subscheme of Affine Space of dimension 2 over Rational Field defined by:
      y,
      x - 1,
     Closed subscheme of Affine Space of dimension 2 over Rational Field defined by:
      x + y + 2,
      2*y^2 + 4*y + 3]
```
seems correct. Here and below.

Reply:
Looks like something went wrong while I resolve the merge conflict… sorry. Will fix soon.